detekt/detekt.bzl

```diff
@@ -70,16 +70,41 @@ detekt = rule(
         # Vendor Detekt CLI.
         # This is not public API yet because ideally Detekt should run as Persistent Worker which will change how we integrate it.
         # Later we should allow user to customize Detekt binary.
-        "_detekt_cli_jar": attr.label(default = "@detekt_cli_jar//file", allow_single_file = True),
-        "_detekt_wrapper_jar": attr.label(default = "//detekt/wrapper:bin_deploy.jar", allow_single_file = True),
-        "srcs": attr.label_list(allow_files = True),
-        "config": attr.label(default = None, allow_single_file = True, doc = "Custom Detekt config file (must end with .yml). If not set Detekt will use its default configuration (mind the Detekt version) https://github.com/arturbosch/detekt/blob/master/detekt-cli/src/main/resources/default-detekt-config.yml"),
+        "_detekt_cli_jar": attr.label(
+            default = "@detekt_cli_jar//file",
+            allow_single_file = True,
+        ),
+        "_detekt_wrapper_jar": attr.label(
+            default = "//detekt/wrapper:bin_deploy.jar",
+            allow_single_file = True,
+        ),
+        "srcs": attr.label_list(
+            mandatory = True,
```

Comment:
I was under impression that without `mandatory = True` and a default value Bazel treats attribute as mandatory, I guess I was wrong!

Reply:
Yep, [the default value is `False`](https://docs.bazel.build/versions/master/skylark/lib/attr.html#label_list).